src/components/Icon.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <svg class="nq-icon" v-html="useTag"></svg>
+</template>
+
+<script lang="ts">
+import {Component, Prop, Vue} from 'vue-property-decorator';
+
+/**
+ * # Icons
+ *
+ * Comment out the ones not used for the build target.
+ *
+ * The call to each imported icon at the end of each line is because the imported
+ * variable has to be used to be processed by webpack. The assignment to _ is to
+ * avoid unused-expression warnings by tslint for this file.
```

Comment:
As far as I can tell, the output is unaffected by these empty assignments. Code-exclusion logic in the minifier removes this dead code.

Reply:
Nice, thanks!